### PR TITLE
chore(dependencies): Bump Quarkus to 1.3.2.Final

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

File: core/core-impl/pom.xml [hunk 1]
```diff
@@ -33,8 +33,8 @@
       <artifactId>commons-text</artifactId>
     </dependency>
     <dependency>
-      <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-smallrye-context-propagation</artifactId>
```
Comment (quintesse) [inline core/core-impl/pom.xml:37]: So Smallrye has become Quarkus-compatible by default?

Comment (gastaldi) [reply core/core-impl/pom.xml:37]: Yes

Comment (gastaldi) [reply core/core-impl/pom.xml:37]: More the opposite (Quarkus become Smallrye compatible) :)

Comment (quintesse) [reply core/core-impl/pom.xml:37]: Aha, ok 👍 😄

**Pull request merged**